content/en/tracing/troubleshooting/tracer_startup_logs.md

```diff
@@ -165,6 +165,8 @@ The Go Tracer prints one of two possible diagnostic lines, one for when the Agen
 {{< /programming-lang >}}
 {{< programming-lang lang="nodejs" >}}
 
+**Note:** Startup logs are disabled by default in version 2.X+. They can enabled using the environment variable `DD_TRACE_STARTUP_LOGS=true`.
```

Comment:
Similar to the above, I would word this as a deprecation instead of a note specific to v2.